src/tactic/lean_core_docs.lean

```diff
@@ -693,3 +693,79 @@ add_tactic_doc
   category   := doc_category.tactic,
   decl_names := [`tactic.interactive.with_cases],
   tags       := ["core", "combinator"] }
+
+/- conv mode tactics -/
```

Comment:
Linking to https://leanprover-community.github.io/extras/conv.html from the docs somehow would be nice, but I have no idea how to do it.

Reply:
Good idea! You could add a docstring to [`tactic.interactive.conv` in core Lean](https://github.com/leanprover-community/lean/blob/master/library/init/meta/converter/interactive.lean#L174) and then it would show up when hovering over the `conv` tactic. Similarly for `conv_lhs` and `conv_rhs`, though those are defined in mathlib's `tactic.converter.interactive`.

Reply:
I think I'll leave that as a follow-up, if that's alright?

Reply:
Yep, fine with me. (I haven't had a chance to read all your new doc strings in detail yet, but I will later today.)